dali/imgcodec/decoders/libtiff/tiff_libtiff.cc

```diff
@@ -241,5 +256,14 @@ DecodeResult LibTiffDecoderInstance::Decode(SampleView<CPUBackend> out, ImageSou
   return {true, nullptr};
 }
 
+DecodeResult LibTiffDecoderInstance::Decode(SampleView<CPUBackend> out, ImageSource *in,
+                                            DecodeParams opts, const ROI &requested_roi) {
```

Comment:
This is already out of date due to API changes. Also, please implement DecodeTaskImpl nistead of DecodeImpl, as it is required by ScheduleDecode.